tests/unit/CMakeLists.txt

```diff
@@ -14,6 +14,8 @@ include_directories(${CMAKE_SOURCE_DIR}/include/system_headers)
 include_directories(${GTEST_SOURCE_DIR}/include)
 include_directories(${GMOCK_SOURCE_DIR}/include)
 
+include_directories(${TCROOT}/i386/gnu-efi/inc)
```

Comment:
This wasn't supposed to go here but in `kernel/arch/x86_64/CMakeLists.txt` which is the CMake file for the 64-bit Tilck kernel. I have a build target for that and has nothing to do with the unit tests. Even if the 64-bit kernel cannot boot because a ton of things are missing, it is built.

Reply:
Oops, I noticed that now with your latest update the target `gcc_8_4_x86_64` succeeds. Maybe I didn't look at the error carefully? I believed that `gcc_8_4_x86_64` was broken because uefi.c couldn't find the header file. What was the problem?

Reply:
I believe it was because the unit tests weren't able to find the gnu-efi header file